express-notes-api/index.js

```diff
@@ -0,0 +1,94 @@
+const fs = require('fs');
+const express = require('express');
+const app = express();
+const data = require('./data');
+const notes = data.notes;
+
+app.use(express.json());
+app.get('/api/notes', (req, res) => {
+  const arr = [];
+  for (const property in notes) {
+    arr.push(notes[property]);
+  }
+  res.json(arr);
+  res.sendStatus(200);
```

Comment:
You want to set the status and send the response on a single line. This is actually an oops it worked. Also by default the status code is `200` so if you want to send a 200 response you don't need to set that. 

You just need:
```JAVASCRIPT
res.json(arr);
```